src/d/actor/d_a_obj_eskban.cpp

```diff
@@ -3,105 +3,451 @@
+    },                                                   // dCcD_SrcGObjInf mObjInf
+    {
+        {
+            {0, 0, 0},  // mCenter
+            200,        // mRadius
+            600,        // mHeight
+        },              // cM3dGCylS mCyl
+    }  // cCcD_SrcCylAttr mCylAttr
+
+};
+static dCcD_SrcCyl cyl_camera_src = {
+    {
+        // objInf
+        {
+            0,  // u32 mFlags
+            {
+                // cCcD_SrcObjHitInf mSrcObjHitInf
+                {
+                    // cCcD_SrcObjAt mObjAt
+                    0,  // u32 mType
+                    0,  // u8 mAtp
+                    {
+                        // cCcD_SrcObjCommonBase mBase
+                        0,  // u32 mSprm
+                    },
+                },
+                {
+                    // cCcD_SrcObjTg mObjTg
+                    0,  // u32 mType
+                    {
+                        // cCcD_SrcObjCommonBase mBase
+                        0,  // u32 mSprm
+                    },
+                },
+                {
+                    // cCcD_SrcGObjCo mObjCo
+                    {
+                        // cCcD_SrcObjCommonBase mBase
+                        CO_SPRM_SET | CO_SPRM_IS_UNK8 | CO_SPRM_VS_UNK8 |
+                            CO_SPRM_NO_CRR,  // u32 mSprm
+                    },
+                },
+            },  // dCcD_SrcGObjInf mObjInf
+        },      // cCcD_SrcObj mObj
+        {},     // dCcD_SrcGObjAt mGObjAt
+        {},     // dCcD_SrcGobjTg mGObjTg
+        {},     // dCcD_SrcGObjCo mGObjCo
+    },
+    {
+        {
+            {0, 0, 0},  // mCenter
+            1000,       // mRadius
+            100,        // mHeight
+        },              // cM3dGCylS mCyl
+    },                  // cCcD_SrcCylAttr mCylAttr
+};
+static dCcD_SrcSph sph_check_src = {
+    {
+        {
+            0,  // u32 mFlags
+            {
+                {},  // cCcD_SrcGObjAt mObjAt
+                {
+                    AT_TYPE_BOMB,  // u32 mType
+                    {
+                        TG_SPRM_SET | TG_SPRM_IS_OTHER,  // u32 mSprm
+                    },                                   // cCcD_SrcObjCommonBase mBase
+                },                                       // cCcD_SrcObjTg mObjTg
+                {
+                    {
+                        CO_SPRM_SET | CO_SPRM_IS_UNK8 | CO_SPRM_VS_UNK2 | CO_SPRM_VS_UNK4 |
+                            CO_SPRM_NO_CRR,  // u32 mSPrm
+                    }  // cCcD_SrcObjCommonBase mBase
+                },  // cCcD_SrcObjCo mObjCo
+            },      // cCcD_SrcObjHitInf mSrcObjHitInf
+        },          // cCcD_SrcObj mObj
+        {},         // dCcD_SrcGObjAt mGObjAt
+        {},         // dCcD_SrcGobjTg mGObjTg
+        {},         // dCcD_SrcGObjCo mGObjCo
+    },
+    {
+        {
+            {0, 0, 0},  // mCenter
+            300,        // mRadius
+        }  // cM3dGSphS mSph
+    }  // cCcD_SrcSphAttr mSphAttr
+};
```

Comment:
I feel like these srcsphs/srccyls are difficult to read with this much nesting and indentation?
I usually put the attributes in a flat list with the field names on the left instead of the right (e.g. in d_a_nh.cpp)

Reply:
this was initially to help me learn the structs but definitely too verbose - I'll follow your example